### DISCUSSION TITLE
Add facebookOauth File

### DISCUSSION THREAD

**Discussion by ambikaaSharma** [datamweb/shield-oauth#37] [Ideas]
Hi @datamweb 

Can you please add facebook aouth file to the "datamweb/shield-oauth/src/Libraries/FacebookOAuth.php" , so that using composer we can update it in our side.

Attached is the zip file, you can extract and place it under path so that we can have latest code at our side ASAP.

It will be very helpful @datamweb 

[FacebookOAuth.zip](https://github.com/datamweb/shield-oauth/files/11249527/FacebookOAuth.zip)

**Comment by datamweb** [comment]
@ambikaaSharma Hello, thanks for sharing.
Why don't you send a PR request to be registered in your name?

**Reply by ambikaaSharma** [reply]
ok will try from next ownwards, Can you please do it once Or  Pull request is the only way @datamweb , Please suggest?

**Reply by datamweb** [reply]
@ambikaaSharma OK, I'll do it, I'm a little busy, it might take a few days. But I recorded it in the task list.

**Reply by ambikaaSharma** [reply]
Ok thanks @datamweb if I create Pull request then how long will it take ?

**Reply by datamweb** [reply]
Before sending PR, see [here](https://github.com/datamweb/shield-oauth/pull/17#issuecomment-1308549694).

If you have done everything well, I will try to check sooner.

**Reply by ambikaaSharma** [reply]
@datamweb  currently I am not aware much about it so I will avoid at this moment and will expect it from your side and will be hppy if it done  ASAP.

Thanks for replying @datamweb really appreciate quick response from your end :-)

**Comment by evansharp** [comment]
I've opened PR #190 with the asset above.
